app/web_hub.go

```diff
@@ -395,10 +445,10 @@ func (h *Hub) Start() {
 				for _, webCon := range candidates {
 					if webCon.ShouldSendEvent(msg) {
 						select {
-						case webCon.Send <- msg:
+						case webCon.send <- msg:
 						default:
 							mlog.Error("webhub.broadcast: cannot send, closing websocket for user", mlog.String("user_id", webCon.UserId))
-							close(webCon.Send)
+							close(webCon.send)
```

Comment:
I see we close the `send` channel in two separate places. Here and in the `directMsg` case above.
Are we safe to do so? Should we add here a check with `connections.Has` like we do above?

Reply:
Yes, definitely. Good point.